package.json

```diff
@@ -36,9 +36,9 @@
     "vitest": "^0.32.2"
   },
   "dependencies": {
-    "@types/libsodium-wrappers-sumo": "^0.7.5",
-    "fast-sha256": "^1.3.0",
-    "get-random-values": "^2.1.0",
-    "libsodium-wrappers-sumo": "^0.7.11"
+    "@noble/ciphers": "^0.1.3",
+    "@noble/hashes": "^1.3.1",
+    "@scure/base": "^1.1.1",
+    "hash-wasm": "^4.9.0"
```

Comment:
maybe set the version of this to "4.9.0" instead of "^4.9.0", for security. Noble can also be set to exact ver, but it's me, while hash-wasm is someone else. Who knows when their account would get hacked?